server/app.py

```diff
@@ -0,0 +1,87 @@
+from flask import Flask
+import requests # https://stackoverflow.com/questions/2018026/what-are-the-differences-between-the-urllib-urllib2-urllib3-and-requests-modul
+import cell_census
+
+# cors
+from flask_cors import CORS, cross_origin
+app = Flask(__name__)
+
+# cors
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
+# open connection to the census
+census = cell_census.open_soma()
+
+@app.route('/')
+def hello_world():
+    return '<h1>This is the "/" route for the python backend for cellxgene-ontology</h1>'
+
+# this route will be hit from a react app running on port 3000
+# it will return a json object
+@app.route('/api')
+@cross_origin(origin='localhost',headers=['Content- Type','Authorization'])
+def api():
+    return {'hello': 'world'}
+
+@app.route('/api/portalDatasets')
+@cross_origin(origin='localhost',headers=['Content- Type','Authorization'])
+def portalDatasets():
```

Comment:
In general, I think any route that does not need to alter it's output based on user arguments should be preloaded and saved to memory at server launch time.

Specific question about this route: Can this metadata be extracted from cell census? Not suggesting we go that route now, but we would probably want to rewrite it to use cell census at some point in the future.

Reply:
Great question. Creating issue. 

https://github.com/chanzuckerberg/ontology-ui/issues/159